### PR TITLE
Add date format option

### PR DESCRIPTION
**Pull request by subahanumanth**

### CODE REVIEW

File: test/pino-roll.test.js [hunk 1]
```diff
@@ -243,3 +243,115 @@ test('symlink rotates on roll', async ({ equal, ok, resolves }) => {
   const content = await readFile(linkPath, 'utf8')
   ok(content.includes('#4'), 'symlink contains fourth log')
 })
+
+test('rotate file with date format based on frequency', async ({ ok, rejects }) => {
```
Comment (mcollina) [inline test/pino-roll.test.js:247]: Can you move those to a different test file? This is becoming too long.

**Comment by mcollina** [comment]
Thanks, just one nit!

File: test/pino-roll.test.js (hunk 1 above)
Comment (subahanumanth) [reply test/pino-roll.test.js]: @mcollina Done!

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**